app/views/courses/TeacherClassView.coffee

```diff
@@ -86,6 +86,12 @@ module.exports = class TeacherClassView extends RootView
 
   initialize: (options, classroomID) ->
     super(options)
+
+    if (options.vue)
```

Comment:
Where do these options come from?

Reply:
These options are passed into the view at creation time and this currently happens in two locations:

1. At construction time in the router (eventually taking options from the call to `go` via `routeDirectly` https://github.com/codecombat/codecombat/blob/school-administrator-dashboard/app/core/Router.coffee#L301
1. When created inside of the teacher class Vue component https://github.com/codecombat/codecombat/blob/school-administrator-dashboard/app/views/teachers/class/TeacherClassView.vue#L6

Reply:
Thanks, looks like we can rename it to something like `fromVueComponent`

Reply:
If I'm understanding properly the `vue` option name is confusing, is that correct?  If so how do you feel about `renderFullPage`?  The `vue` option only controls if we render the page content or page content and page template.

Reply:
Yep that also sounds fine, `renderFullPage` or `renderWithNav` ?

Reply:
Took a look at this again and the option determines if we render only content or not.  I changed to `renderOnlyContent`.  I can also see an argument for `renderWithoutPageLayout`?

Reply:
Let me know what you think